XCUITests/ToolbarTest.swift

```diff
@@ -96,6 +96,28 @@ class ToolbarTests: BaseTestCase {
         XCTAssertEqual(value as? String, "", "The url has not been removed correctly")
     }
 
+    //Check that that after scrolling on a page, the URL bar is hidden. Tapping one on the status bar will reveal the URL bar, tapping again on the status will scroll to the top
+    func testRevealToolbarWhenTappingOnStatusbar(){
+        //Workaround when testing on iPhone. If the orientation is in landscape on iPhone the tests will fail.
+        if !iPad() {
+            XCUIDevice.shared.orientation = UIDeviceOrientation.portrait
+        }
+        waitforExistence(app/*@START_MENU_TOKEN@*/.buttons["HomePanels.TopSites"]/*[[".otherElements[\"Panel Chooser\"]",".buttons[\"Top sites\"]",".buttons[\"HomePanels.TopSites\"]"],[[[-1,2],[-1,1],[-1,0,1]],[[-1,2],[-1,1]]],[0]]@END_MENU_TOKEN@*/)
```

Comment:
I guess you got that long and not really easy to read element by recording the test...please try to write it as it is shown but avoiding the blue highlighting the text:
`app.buttons["HomePanels.TopSites"]` and same for the pageOptionButton, that way the code will look cleaner in the repo...